views/menus/stav.js

```diff
@@ -4,67 +4,199 @@ import React from 'react'
 import {
   StyleSheet,
   View,
-  NetInfo,
-  WebView,
+  ListView,
+  Text,
+  ActivityIndicator,
 } from 'react-native'
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-  },
-  webView: {
-    flex: 1,
-  },
-})
+import FoodItem from './foodItem'
+
+import moment from 'moment-timezone'
+const CENTRAL_TZ = 'America/Winnipeg'
 
-const URL = 'http://stolaf.cafebonappetit.com/cafe/stav-hall#Lunch'
-// const DEFAULT_URL = 'http://legacy.cafebonappetit.com/print-menu/cafe/261/menu/112732/days/today/pgbrks/0/'
+import DietaryFilters from './dietaryFilters'
+const dietaryFilters = DietaryFilters()
+
+import buildingHours from '../../data/building-hours.json'
 
 export default class StavMenuView extends React.Component {
-  state = {
```

Comment:
Ah yes, apologies. I looked for that but didn't see it. Now I do. Criticism withdrawn!